src/modules/memory.cpp

```diff
@@ -18,19 +18,31 @@ namespace modules {
   memory_module::memory_module(const bar_settings& bar, string name_) : timer_module<memory_module>(bar, move(name_)) {
     m_interval = m_conf.get<decltype(m_interval)>(name(), "interval", 1s);
 
-    m_formatter->add(DEFAULT_FORMAT, TAG_LABEL, {TAG_LABEL, TAG_BAR_USED, TAG_BAR_FREE, TAG_RAMP_USED, TAG_RAMP_FREE,
+    m_formatter->add(DEFAULT_FORMAT, TAG_LABEL, {TAG_LABEL, TAG_BAR_AVAILABLE_USED, TAG_BAR_AVAILABLE_FREE, TAG_BAR_FREE_USED, TAG_BAR_FREE, TAG_RAMP_AVAILABLE_USED, TAG_RAMP_AVAILABLE_FREE, TAG_RAMP_FREE_USED, TAG_RAMP_FREE,
```

Comment:
This single line is more than 200 characters long. It's harder to read and understand. Maybe split it into 3-4 lines?